src/SEnergyCorrelatorConfig.h

```diff
@@ -49,6 +49,11 @@ namespace SColdQcdCorrelatorAnalysis {
     float pTSmear = 0;
     float theta = 0;
 
+    //Manual Calculation option
+    bool doManualCalc = false;
+    int mom_option = 0;
+    int norm_option = 0;
```

Comment:
Like above, you can use the enum itself here:

```suggestion
    int mom_option = SEnergyCorrelator::Norm::E;
    int norm_option = SEnergyCorrelator::Norm::E;
```